plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -740,4 +740,13 @@ private void updateAllDependents(String parent, Plugin dependent, Map<String,Ver
             }
         }
     }
+
+    private Map<String, VersionNumber> difference(Map<String, VersionNumber> base, Map<String, VersionNumber> toAdd) {
```

Comment:
Maybe "addDifference"?  And I'll add an annotation to the method; usually I do sorry I missed this one.